fs2-google-pubsub-http/src/main/scala/com/permutive/pubsub/consumer/http/PubsubHttpConsumer.scala

```diff
@@ -28,10 +31,11 @@ object PubsubHttpConsumer {
     serviceAccountPath: String,
     config: PubsubHttpConsumerConfig[F],
     httpClient: Client[F],
-    errorHandler: (PubsubMessage, Throwable, F[Unit], F[Unit]) => F[Unit]
+    errorHandler: (PubsubMessage, Throwable, F[Unit], F[Unit]) => F[Unit],
+    httpClientRetryPolicy: RetryPolicy[F] = recklesslyRetryPolicy[F]
```

Comment:
How about we dont set it as default arg, just to minimize surprise.

People can upgrade and set their own if they need to

Reply:
Since the retry mechanism is actually fixing a bug, I think users should start using it by next release by default.

But I understand your point as well, and if we go the no surprise way, we should somehow make the users know that there's a default retry policy they can use and where to find it.

Let us know what you think as well @CremboC @mcgizzle

Reply:
That's a documentation problem, I think it is typically solved by writing release note?

tbh I don't feel super strongly personally since we (modeling-team) want this anyway, I am just that a change in operational behavior of a library should be an explicit choice of library user

Reply:
I see your point, do you think this is true even if the change in operational behaviour fixes a bug?

(You are kind of convincing me 😛)